SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_006004	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, increases the damage dealt on an enemy by 35 per attribute level.
+SKILL_20151224_006005	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, it will continuously apply the [Attacked Weakened] status effect. Increases the duration by 8 seconds per attribute level and [Attacked Weakened] can stack up to 10 times.
+SKILL_20151224_006006	Evocation: Enhance
+SKILL_20151224_006007	Increases the damage dealt on an enemy with [Evocation] by 1% per attribute level.
+SKILL_20151224_006008	Desmodus: Enhance
+SKILL_20151224_006009	Increases the damage dealt on an enemy with [Desmodus] by 1% per attribute level.
+SKILL_20151224_006010	Critical hit resistance of the enemy fallen to [Slow] is decreased by 15% of the user's intelligence value.
+SKILL_20151224_006011	Sonic Strike: Enhance
+SKILL_20151224_006012	Increases the damage dealt on an enemy with [Sonic Strike] by 1% per attribute level.
+SKILL_20151224_006013	Increase your accuracy by 5% per attribute level when attacking with gun equipments.
+SKILL_20151224_006014	Wild Shot: Enhance
+SKILL_20151224_006015	Increases the damage dealt on an enemy with [Wild Shot] by 1% per attribute level.
+SKILL_20151224_006016	Cannon Mastery: Penetration
+SKILL_20151224_006017	Increases your block penetration by 6% per attribute level when attacking with Cannoneer skills.
+SKILL_20151224_006018	Airborne enemy hit by [Showdown] is grounded for 1.8 seconds per attribute level.
+SKILL_20151224_006019	Increases attacking range of [Siege Burst] by 25 per attribute level.
+SKILL_20151224_006020	Increases the chance to instant-kill small type enemy with [Rifle] by 0.25% per attribute level. (Does not apply if the small type enemy is higher level than the character level.)
+SKILL_20151224_006021	Fire resistance is increased by 14 per attribute level when [Aukuras] is active.
+SKILL_20151224_006022	Magic amplification is increased by 12 per attribute level when [Zalciai] is active.
+SKILL_20151224_006023	Comrade's evasion is increased by 2% per attribute level when [Monstrance] is active.
+SKILL_20151224_006024	Increases the physical defense of the zombies while using [Bwa Kayiman] by 8 per attribute level.	
+SKILL_20151224_006025	Increases the body's evasion by 10% per attribute level while [Out of Body] is active.
+SKILL_20151224_006026	Deals damage equal to 20% magic attack per attribute level to enemies hit with [Barrier].
+SKILL_20151224_006027	Increases the character's SP recovery. Increases the SP recovery by 10 per attribute level.
+SKILL_20151224_006028	Damage the enemy when they step on the magic circle created by [Healing Factor]. The damage is determined by the value of current HP.
+SKILL_20151224_006029	The range of [Fumigate] becomes a purification area for 10 seconds. Increases the comrade's poison resistance by 10% per attribute level.
```

Comment:
Increases the comrade's poison resistance by 10% per attribute level.

to

 Increases the ally's poison resistance by 10% per attribute level.